swagger-spec/metaschemas/list.yaml

```diff
@@ -1 +1,382 @@
 # /metaschemas/
+get:
+  summary: List all metaschemas
+  description: >-
+
+    A paginated list of all active metaschemas.
+
+    Metaschemas describe the supplemental questions that accompany a registration.
+
+    #### Returns
+
+    Returns a JSON object containing `data` and `links` keys.
+
+
+    The `data` key contains an array of 10 metaschemas.
+    Each resource in the array is a separate metaschema object.
+
+
+    The `links` key contains a dictionary of links that can be used
+    for [pagination](#Introduction_pagination).
+
+
+    This request should never return an error.
+
+  tags:
+    - Metaschemas
+  operationId: metaschemas_list
+  x-response-schema: Metaschema
+  responses:
```

Comment:
In the interest of the example response section not being ridiculously long (at least until we have scrolling in that section of the docs) can you limit this response to maybe just the simplest metaschema? I think that would be either the "Open-ended Registration" or "OSF-Standard Pre-Data Collection Registration" metaschema.